<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Insert</class>
 <widget class="QDialog" name="Insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(121, 121, 121);
border-radius:20px  ;
}
QLineEdit{
border-radius:6px  ;
}
QPushButton{
background-color: rgb(242, 255, 210);
background-color: rgb(255, 170, 255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;	
border-radius:3px  ;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>23</number>
    </property>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>ISBN</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="placeholderText">
       <string>书名</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="placeholderText">
       <string>作者</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>出版时间</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="placeholderText">
       <string>出版社</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>库存总数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>录入</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
